pattern_localization/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 project( pattern_localizer )
 
-find_package(OpenCV REQUIRED)
+find_package(OpenCV REQUIRED PATHS /home/anikde/Documents/3rd_party/openCV/opencv/install/lib/cmake/opencv4 NO_DEFAULT_PATH)
```

Comment:
This path is specific to your system and will not work anywhere else.
That is why CI fails to build.